lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -74,6 +75,12 @@ def permalink_title
                                end
         end
 
+        def permalink
+          # Hpricot thinks "link" is a self closing tag so it puts the text of the link after open tag
+          uri = text_for('link') || @node.at('link').following[0].to_s
+          @permalink ||= URI(uri).path
```

Comment:
When you memoize like this, every call to method `:permalink` resets variable `uri` unnecessarily.
You should either not memoize this method, or memoize early to assign the variable `uri` just once:
```ruby
def permalink
  @permalink ||= begin
    uri = text_for('link') || @node.at('link').following[0].to_s
    URI(uri).path
  end
end
```